src/components/MainComponent.js

```diff
@@ -3,9 +3,9 @@ import { Provider } from 'react-redux'
 import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'
 import getMuiTheme from 'material-ui/styles/getMuiTheme'
 import { Router, Route } from 'react-router-dom'
-import createBrowserHistory from 'history/createBrowserHistory'
 import { PersistGate } from 'redux-persist/lib/integration/react'
 
+import createBrowserHistory from 'history/createBrowserHistory'
```

Comment:
I think this is a third party library, not sure why it was moved to the first party imports section.